Server/src/controllers/authenticationController.ts

```diff
@@ -49,6 +49,17 @@ export class AuthenticationController {
     }
   }
 
+  async createLogoutRequest(request: Request, response: Response, next: NextFunction) {
+    const tokenValue = request.cookies && request.cookies.token;
+    console.log('token: ', tokenValue)
+    response.cookie('token', 'tokenValue', { path: '/' });
+    response.clearCookie('token', { path: '/' });
```

Comment:
so, the first one sets the token, the clearCookie probably clears it, do ;you need the`response.cookie('token', 'tokenValue', { path: '/' });`?

Reply:
according to expressJS docs, it says i have to? not entirely sure though
![image](https://user-images.githubusercontent.com/25268105/107131681-b5452700-68a6-11eb-90fe-31d2b5d5d290.png)

Reply:
oh i see, so you're saying it doesnt work without that line for you?

Reply:
i removed it and its still working. i guess you're right in this case